app/views/passengers/_filters_and_key.html.haml

```diff
@@ -12,6 +12,7 @@
       = f.submit 'Print'
       -if @current_user.admin?
         %input.btn.btn-secondary#copybtn{type: 'button', value: 'Copy Emails'}
+        = link_to('Export CSV', passengers_path(format: 'csv', status: params[:status]), class: 'btn btn-secondary')
```

Comment:
Can I perhaps point you to [this comment](https://github.com/umts/st-pax/issues/404#issuecomment-2023941484) as a hint? A form can have more than one submit button — in this case one for PDF, one for CSV.